examples/notebooks/beam-ml/run_inference_tensorflow_hub.ipynb

```diff
@@ -0,0 +1,307 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "634de4ce-07f7-4003-8ff6-d3f58b41a79d"
+      },
+      "source": [
+        "##### Copyright 2022 Google Inc.\n",
```

Comment:
I think we should remove the Google Inc license and keep Apache license only.

Reply:
For example, like this notebook: https://github.com/apache/beam/blob/3de2f85b3439a7084b60eb600a56bdc4cd36525c/examples/notebooks/beam-ml/run_inference_pytorch.ipynb

Reply:
+1, I don't think we're allowed to accept a google license into the code base

Reply:
I think this is what we want for the license cell section:

{
  "cells": [
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "634de4ce-07f7-4003-8ff6-d3f58b41a79d"
      },
      "outputs": [],
      "source": [
        "# @title ###### Licensed to the Apache Software Foundation (ASF), Version 2.0 (the \"License\")\n",
        "\n",
        "# Licensed to the Apache Software Foundation (ASF) under one\n",
        "# or more contributor license agreements. See the NOTICE file\n",
        "# distributed with this work for additional information\n",
        "# regarding copyright ownership. The ASF licenses this file\n",
        "# to you under the Apache License, Version 2.0 (the\n",
        "# \"License\"); you may not use this file except in compliance\n",
        "# with the License. You may obtain a copy of the License at\n",
        "#\n",
        "#   http://www.apache.org/licenses/LICENSE-2.0\n",
        "#\n",
        "# Unless required by applicable law or agreed to in writing,\n",
        "# software distributed under the License is distributed on an\n",
        "# \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY\n",
        "# KIND, either express or implied. See the License for the\n",
        "# specific language governing permissions and limitations\n",
        "# under the License"
      ]
    },

You can see an example here: https://github.com/apache/beam/blob/master/examples/notebooks/beam-ml/custom_remote_inference.ipynb?short_path=ad25849#L3

Reply:
Updated license.